docs/background/energy_offset_array.rst

```diff
@@ -0,0 +1,24 @@
+.. _bg_models:
+
+EnergyOffset Array
+=================
```

Comment:
Please fix the Sphinx warnings:
```
$ python setup.py build_sphinx -l
...
/Users/deil/code/gammapy/docs/background/energy_offset_array.rst:4: WARNING: Title underline too short.

EnergyOffset Array
=================
/Users/deil/code/gammapy/docs/background/energy_offset_array.rst:11: ERROR: Unexpected indentation.
/Users/deil/code/gammapy/docs/background/models.rst:4: WARNING: duplicate label bg_models, other instance in /Users/deil/code/gammapy/docs/background/energy_offset_array.rst
...
```
Let me know if I should explain what they mean and how to fix them.